config/config/sc-config.yaml

```diff
@@ -159,9 +176,21 @@ elasticsearch:
       limits:
         memory: 1024Mi
         cpu: 1
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
+          podAffinityTerm:
+            topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                role: master
```

Comment:
This would clash with pods from other applications right? If we have any other pods that would have this label.
So maybe add `app: opendistro-es` as well to prevent this (not sure this is needed right now, mostly future proofing).

Reply:
I'd rather not change this. I only want to get more config options with this pr and not nessecarily improve the current configration values.

Reply:
Ok, sure :)